cmd/xpumanager_sidecar/README.md

```diff
@@ -64,11 +65,73 @@ $ kubectl patch daemonsets.apps intel-xpumanager --patch-file 'https://github.co
 
 NOTE: The sidecar patch will remove other resources from the XPU-Manager container. If your XPU-Manager daemonset is using, for example, the smarter device manager resources, those will be removed.
 
-#### Verify Sidecar Functionality
+### Verify Sidecar Functionality
 
 You can verify the sidecar's functionality by checking node's xe-links labels:
 
 ```bash
 $ kubectl get nodes -A -o=jsonpath="{range .items[*]}{.metadata.name},{.metadata.labels.gpu\.intel\.com\/xe-links}{'\n'}{end}"
 master,0.0-1.0_0.1-1.1
 ```
+
+### Use HTTPS with XPU-Manager
+
+XPU-Manager can be configured to use HTTPS on the metrics interface. For the gunicorn sidecar, cert and key files have to be added to the command:
+```
+      - command:
+        - gunicorn
+...
+        - --certfile=/certs/tls.crt
+        - --keyfile=/certs/tls.key
+...
+        - xpum_rest_main:main()
+```
+
+The cert and key files have to be included in the container by a secret. Certificate and keys can be added manually or by leveraging [cert-manager](https://cert-manager.io/).
+
+<details>
+<summary>Example for the Cert-manager objects</summary>
+
+```
+apiVersion: cert-manager.io/v1
+kind: Issuer
+metadata:
+  name: selfsigned-issuer
+spec:
+  selfSigned: {}
+---
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: serving-cert
+spec:
+  dnsNames:
+  - xpum.svc
+  - xpum.svc.cluster.local
+  issuerRef:
+    kind: Issuer
+    name: selfsigned-issuer
+  secretName: xpum-server-cert
+```
+One would then include `xpum-server-cert` in the XPU-manager deployment.
```

Comment:
As this part is not specific to using `cert-manager` for generating the secrets, I do not think it should be hidden under (same) `<details>`.

IMHO better would be to have it earlier, maybe in same example with `gunicorn` args:
```
    volumes:
    - name: certs
      secret:
        defaultMode: 420
        secretName: xpum-server-cert
...
    containers:
      - name: python-exporter
        volumeMounts:
        - mountPath: /certs
          name: certs
          readOnly: true
...
      - command:
        - gunicorn
...
        - --certfile=/certs/tls.crt
        - --keyfile=/certs/tls.key
...
        - xpum_rest_main:main()
```